flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/JsonBatchFileSystemITCase.java

```diff
@@ -36,10 +36,14 @@
 import java.util.Arrays;
 import java.util.List;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 /** ITCase to test json format for {@link JsonFormatFactory}. */
-public class JsonBatchFileSystemITCase extends BatchFileSystemITCaseBase {
+class JsonBatchFileSystemITCase extends BatchFileSystemITCaseBase {
```

Comment:
given that the BatchFileSystemITCaseBase hasn't been migrated to junit4 (and thus a lot of things from it won't work), does this test actually behave as expected?

Reply:
yes,it test actually behave as expected
<img width="1459" alt="image" src="https://user-images.githubusercontent.com/20968409/165985869-4a2ff329-176b-44c4-96e5-10c658a1ae2f.png">

Reply:
"Running" and "behaving as expected" are 2 different things. For example. the BatchAbstractTestBase sets up a mini cluster against which the test should be run. Is this being honored?

Reply:
ok, i now change it back to junit4 and accept your suggestion to apply assertThat